core/XH.js

```diff
@@ -324,9 +324,20 @@ class XHClass extends HoistBase {
         return this.acm.themeModel.toggleTheme();
     }
 
-    /** Enable/disable the dark theme directly (useful for custom app option controls). */
+    /**
+     * Enable/disable the dark theme directly (useful for custom app option controls).
+     * @param {boolean} value
+     */
     setDarkTheme(value) {
```

Comment:
Think we should deprecate or remove `setDarkTheme()` and make "setTheme()" (without the suffix) the main entry point.  It's too confusing, and the 'pref' bit is an implementation detail.